ur_robot_driver/include/ur_robot_driver/hardware_interface.hpp

```diff
@@ -76,24 +75,20 @@ class URPositionHardwareInterface : public hardware_interface::SystemInterface
 public:
   RCLCPP_SHARED_PTR_DEFINITIONS(URPositionHardwareInterface);
 
-  return_type configure(const HardwareInfo& system_info) final;
+    CallbackReturn on_init(const HardwareInfo& system_info) final;
 
   std::vector<hardware_interface::StateInterface> export_state_interfaces() final;
 
   std::vector<hardware_interface::CommandInterface> export_command_interfaces() final;
 
-  status get_status() const final
-  {
-    return status_;
-  }
-
   std::string get_name() const final
   {
     return info_.name;
   }
```

Comment:
This should be now implemented in the [default implementation](https://github.com/ros-controls/ros2_control/blob/04fa19b50e9e67cf7e6d7cc5c7ba000e435440f9/hardware_interface/include/hardware_interface/system_interface.hpp#L183). It is safe to delete.

```suggestion
```

Reply:
d4a1834